gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStreamIntegrationTest.java

```diff
@@ -267,21 +267,17 @@ public void operation_durationMetric_tests() throws Exception {
     assertThat(in.read(1, value, 1, 1)).isEqualTo(1);
     assertThat(value).isEqualTo(expected);
 
-    // One seek is to seek to the position and another seek to seek back to the original position.
-    TestUtils.verifyCounter(stats, STREAM_READ_SEEK_OPERATIONS, 2);
+    TestUtils.verifyCounter(stats, STREAM_READ_SEEK_OPERATIONS, 0);
     TestUtils.verifyCounter(stats, STREAM_READ_SEEK_BACKWARD_OPERATIONS, 2);
 
-    TestUtils.verifyDurationMetric(stats, STREAM_READ_OPERATIONS, 2);
+    TestUtils.verifyDurationMetric(stats, STREAM_READ_OPERATIONS, 0);
 
     in.seek(0);
 
-    TestUtils.verifyDurationMetric(stats, STREAM_READ_SEEK_OPERATIONS, 3);
-
-    TestUtils.verifyDurationMetric(stats, STREAM_READ_CLOSE_OPERATIONS, 1);
+    TestUtils.verifyDurationMetric(stats, STREAM_READ_SEEK_OPERATIONS, 0);
 
-    TestUtils.verifyDurationMetric(stats, STREAM_READ_CLOSE_OPERATIONS, 2);
```

Comment:
why we have removed close operation  verification?

Reply:
Same as the other instance. There was a bug in the test verification. There is only 1 close operation and it is checked below.